.github/workflows/ci.yml

```diff
@@ -73,32 +75,37 @@ jobs:
 
         - name: Node.js 8.x
           node-version: "8.17"
-          npm-i: mocha@7.2.0
+          npm-i: mocha@7.2.0 nyc@14.1.1
 
         - name: Node.js 9.x
           node-version: "9.11"
-          npm-i: mocha@7.2.0
+          npm-i: mocha@7.2.0 nyc@14.1.1
 
         - name: Node.js 10.x
-          node-version: "10.23"
+          node-version: "10.24"
+          npm-i: mocha@8.4.0
 
         - name: Node.js 11.x
           node-version: "11.15"
+          npm-i: mocha@8.4.0
 
         - name: Node.js 12.x
-          node-version: "12.20"
+          node-version: "12.22"
 
         - name: Node.js 13.x
           node-version: "13.14"
 
         - name: Node.js 14.x
-          node-version: "14.15"
+          node-version: "14.18"
 
         - name: Node.js 15.x
-          node-version: "15.5"
+          node-version: "15.14"
 
         - name: Node.js 16.x
-          node-version: "16.2"
+          node-version: "16.13"
+
+        - name: Node.js 17.x
+          node-version: "17.2"
```

Comment:
Can we move this to a separate PR? Or include up to 21 in this one? I will suggest to keep the PR the most atomic possible and avoid add new Node.js versions in this fix.

Reply:
I will move it to a separate PR

Reply:
removed

Reply:
PR adding new versions https://github.com/jshttp/statuses/pull/32